README.md

```diff
@@ -148,7 +148,43 @@ For non-Java applications or tools like `cqlsh`, you can run the Spanner Cassand
 
     This will create an executable jar file `spanner-cassandra-launcher.jar` inside the folder `google-cloud-spanner-cassandra/target`.
 
-* Run the jar using the command:
+* Run the jar using one of the following methods:
```

Comment:
could you also add `config_options.md` under `docs/` similar to https://github.com/cloudspannerecosystem/cassandra-to-spanner-proxy/blob/main/docs/config_options.md and add link here